applications/openshift/kubelet/kubelet_enable_streaming_connections/rule.yml

```diff
@@ -31,7 +36,7 @@ ocil_clause: 'the streaming connection timeouts are not disabled'
 
 ocil: |-
     Run the following command on the kubelet node(s):
-    <pre>$ sudo grep streamingConnectionIdleTimeout {{{ kubeletconf_path }}}</pre>
+    <pre>$ oc get --raw /api/v1/nodes/${NODE_NAME}/proxy/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubelet.config.k8s.io/v1beta1"' | grep streamingConnectionIdleTimeout</pre>
     The output should return <tt>{{{ xccdf_value("var_streaming_connection_timeouts") }}}</tt>.
```

Comment:
Not related to this patch and maybe I can take a look as part of the focusing, but do you know off-hand why is the variable not rendered in the CCR? I'm getting:
```
Id:            xccdf_org.ssgproject.content_rule_kubelet_enable_streaming_connections
Instructions:  Run the following command on the kubelet node(s):
$ oc get --raw /api/v1/nodes/${NODE_NAME}/proxy/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubelet.config.k8s.io/v1beta1"' | grep streamingConnectionIdleTimeout
The output should return .
Kind:  ComplianceCheckResult
```
Note the space after "should return"

Reply:
And thinking about it..maybe we should spell out the variable name verbatim so that it's even clearer that a variable is used, what do you think? Something like `make sure the variable matches the value of the variable var-streaming-connection-timeouts, defaulting to <tt>{{{ xccdf_value("var_streaming_connection_timeouts") }}}</tt>`

Reply:
Ok I see why, because we should use `{{.var_streaming_connection_timeouts}}` there to get it rendered in CCR